packages/wrangler/e2e/dev.test.ts

```diff
@@ -830,7 +830,7 @@ describe("analytics engine", () => {
 						"wrangler.toml": dedent`
 				name = "${workerName}"
 				main = "src/index.ts"
-				compatibility_date = "2024-08-08"
+				compatibility_date = "2022-08-08"
```

Comment:
why this change?

Reply:
This is a compat date before JSRPC was enabled, and so should test that AE works with the old compat date

Reply:
Oh I see - the regression was triggered by old compat dates - so this is checking that.